src/modules/commander/Commander.cpp

```diff
@@ -2151,6 +2159,8 @@ Commander::run()
 						PX4_ERR("Engine Failure");
 						set_health_flags(subsystem_info_s::SUBSYSTEM_TYPE_MOTORCONTROL, true, true, false, status);
 					}
+				} else {
+					_engine_thrust_high = (throttle > ef_throttle_thres) && (current2throttle > ef_current2throttle_thres);
```

Comment:
Is this being used?

Reply:
Left over from previous method being evaluated. Will remove.